### DISCUSSION TITLE
How to represent a density matrix and reduced density matrix for certain sites after gaining a variational state?

### DISCUSSION THREAD

**Discussion by Dzhou12** [netket/netket#1505] [Q&A]
I'm now trying to use residual entropy to characterize the order of many body system constitutes of spin 1/2s  and I don't know how to get the density matrix of the system and further the reduced matrix of certain two spins in netket after I gain the variational state after training.

**Comment by PhilipVinc** [comment]
Hi @Dzhou12 , could you please be more specific about what you would like to compute, maybe writing down the formulas, and what estimators/algorithms you would like to use?
